repose-aggregator/tests/functional-tests/src/integrationTest/groovy/features/core/proxy/ContentLengthTest.groovy

```diff
@@ -64,7 +65,33 @@ class ContentLengthTest extends ReposeValveTest {
         "PUT"   | null    | 0            | 1              | 0
         "TRACE" | "blah"  | 1            | 0              | 0
         "TRACE" | null    | 0            | 0              | 0
-
     }
 
+    @IgnoreIf({ new Date() < timeBombDate.getTime() })
```

Comment:
Yeah, I think we use the jUnit `Assume` elsewhere, but since this is in a Spock test, I kind of wanted to use a Spock construct. So here it is.